db/version_set.cc

```diff
@@ -1960,7 +1962,8 @@ void Version::MultiGet(const ReadOptions& read_options, MultiGetRange* range,
       return;
     }
     uint64_t batch_size = 0;
-    for (auto iter = file_range.begin(); iter != file_range.end(); ++iter) {
+    for (auto iter = file_range.begin(); s.ok() && iter != file_range.end();
```

Comment:
Nit: No need to check ```Status``` here since there is a ```break``` on line 2040.

Reply:
The break on line 2040 exits the outer loop (for different sst files) and this one is for inner loop that iterates over keys in one sst file.

Reply:
Oh that's right! Sorry, misread the code.